.github/workflows/publish.yml

```diff
@@ -150,7 +150,7 @@ jobs:
       - name: Scan ${{ matrix.image }} for vulnerabilities
         uses: aquasecurity/trivy-action@master
         with:
-          image-ref: ${{ env.REGISTRY }}/${{ env.OWNER }}/${{ matrix.image }}:develop
+          image-ref: ${{ env.REGISTRY }}/${{ env.OWNER }}/${{ matrix.image }}:pr-1543
```

Comment:
Note: this change needs to be reverted before merging

Reply:
👍 done & squashed out